lib/knackhq/client.rb

```diff
@@ -0,0 +1,42 @@
+require 'knackhq/client/version'
+require 'blanket'
+require 'hashie'
+
+module Knackhq
+  class Client
+    attr_accessor :base_uri, :x_knack_application_id, :x_knack_rest_api_key
+
+    def initialize(base_uri, x_knack_application_id, x_knack_rest_api_key)
+      @base_uri = base_uri
+      @x_knack_application_id = x_knack_application_id
+      @x_knack_rest_api_key = x_knack_rest_api_key
+    end
+
+    def objects
```

Comment:
Blanket works as the following syntax:
```Ruby
github.users('inf0rmer').repos.get
```

I was thinking of using the same method to get objects IE
```Ruby
client = module Knackhq::Client.new(attributes here...)
client.object_1
#returns type ClientQuery hash value
#<ClientQuery  { :name => 'name', :id => 1 } >
client.objects
#<ClientQuery [ { :name => 'name', :id => 1 },  { :name => 'name2', :id => 2 } ]
client.err
# We can discuss what it returns:
# <ClientQuery method_missing? >
```